lib/pure/ioselects/ioselectors_select.nim

```diff
@@ -451,3 +451,6 @@ template withData*[T](s: Selector[T], fd: SocketHandle|int, value,
     else:
       body2
 
+
+proc getFd*[T](s: Selector[T]): int =
```

Comment:
Shouldn't this be a ``SocketHandle``?

Reply:
A selector isn't a Socket. Again, I go back to the fact that this type is a misnomer. Perhaps I should rename it?

Reply:
Rename it then, yes.